build/Build.Steps.cs

```diff
@@ -309,9 +309,37 @@ partial class Build
                 .SetFramework(TargetFramework.NET6_0)
                 .SetOutput(TracerHomeDirectory / MapToFolderOutput(TargetFramework.NET6_0)));
 
+            Log.Debug("Remove files available in additional store from net folder");
```

Comment:
maybe add some more encapsulation? this makes the function huge and basically has no description (eg: function name) what is the huge block doing.

Reply:
3bd5e8f573ed68d0787db42d278c75416e966fb4